### PR TITLE
[4.X] workflow filter

### PR DESCRIPTION
**Pull request by niharikamahajan02**
Pull Request for Issue #36802

### Summary of Changes
An option to select one (or more) workflow stages to filter articles in the admin module.

### CODE REVIEW

**Comment by niharikamahajan02** [comment]
Actually it is not the final commit as one problem is how do I get all workflow options in dropdown

**Comment by brianteeman** [comment]
The field should only be displayed if workflows is enabled

**Comment by niharikamahajan02** [comment]
Yes ok , would do that .

File: administrator/modules/mod_latest/src/Helper/LatestHelper.php [hunk 1]
```diff
@@ -64,6 +64,13 @@ public static function getList(Registry &$params, ArticlesModel $model)
 		{
 			$model->setState('filter.category_id', $categoryId);
 		}
+  //set workflow filter
```
Comment (richard67) [inline administrator/modules/mod_latest/src/Helper/LatestHelper.php]:
```suggestion
		//set workflow filter
```

File: administrator/modules/mod_latest/src/Helper/LatestHelper.php [hunk 2]
```diff
@@ -64,6 +64,13 @@ public static function getList(Registry &$params, ArticlesModel $model)
 		{
 			$model->setState('filter.category_id', $categoryId);
 		}
+		//set workflow filter
```
Comment (richard67) [inline administrator/modules/mod_latest/src/Helper/LatestHelper.php]:
```suggestion

		// Set workflow filter
```
Sorry, I've missed the missing empty line to be added for code style.

Comment (chmst) [reply administrator/modules/mod_latest/src/Helper/LatestHelper.php]: Comments start with a blank after th // and a capital letter. Please compare other comments.

Comment (richard67) [reply administrator/modules/mod_latest/src/Helper/LatestHelper.php]: That's right. How could I miss that? Will adjust my suggestion.

File: administrator/modules/mod_latest/mod_latest.xml [hunk 3]
```diff
@@ -66,6 +66,22 @@
...
+					extension="com_content.article"
+					
+					default=""
+					filter="integer"
+				
+					>
+					
+				<option value="">JOPTION_ANY_CATEGORY</option>
+					
+				</field	>
```
Comment (brianteeman) [inline administrator/modules/mod_latest/mod_latest.xml:83]: Please look at how all the other fields are formatted

**Comment by brianteeman** [comment]
Looks like this has been abandoned in a very early state. If so theen it should be closed and the original issue reopened.

**Comment by brianteeman** [comment]
Why is this abandoned pr still open?

**Comment by HLeithner** [comment]
This pull request has automatically rebased to 4.2-dev.

**Comment by Quy** [comment]
@niharikamahajan02 Closing as it appears to be abandoned.  If this is not the case, please advise. Thanks for contributing.

**Pull request closed without merging**